android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java

```diff
@@ -106,6 +109,12 @@ public void onCreate() {
         }
         refWatcher = LeakCanary.install(this);
 
+        //Initialize Cache
+        File httpCacheDirectory = new File(getCacheDir(), "picasso-cache");
+        Cache cache = new Cache(httpCacheDirectory, 15 * 1024 * 1024);
+
```

Comment:
this is the amount of memory the OS will allocate to the process right after the app opens?
are we sure OS will always provide this amount of cache?
Also if no, then dont you think rendering images will become a problem?